worker/src/RTC/PipeTransport.cpp

```diff
@@ -388,6 +388,7 @@ namespace RTC
 						MS_THROW_TYPE_ERROR("missing port");
 					}
 
+					// NOLINTNEXTLINE (bugprone-unchecked-optional-access)
```

Comment:
The reason why I add this label here rather than doing as in the next cases is because the variable `port` is already taken and cannot be reused (different types). And it feels a bit dirty naming it differently. In this case the label means: "we know what we are doing (TM)"